lib/mayaUsd/ufe/UsdAttributes.cpp

```diff
@@ -504,28 +504,37 @@ bool UsdAttributes::canRenameAttribute(
 
 static void setConnections(
     const PXR_NS::UsdPrim& prim,
-    const SdfPath&         OldPropertyPath,
+    const SdfPath&         oldPropertyPath,
```

Comment:
The method does not set the connection it only updates existing connections:
`updateConnections()`